iceoryx_examples/callbacks/ice_callbacks_subscriber.cpp

```diff
@@ -44,7 +44,10 @@ void shutdownTriggerCallback(iox::popo::UserTrigger*)
 
 void onSampleReceived(iox::popo::Subscriber<CounterTopic>* subscriber)
 {
-    subscriber->take().and_then([](auto& sample) { printf("received: %d\n", sample->counter); });
+    subscriber->take().and_then([](auto& sample) {
+        printf("received: %d\n", sample->counter);
```

Comment:
I guess printf() was here on purpose because it prints the whole string at once into the console. std::cout prints in multiple parts, e.g. `std::cout << m_var1 << "mytext std::endl;" is printing first m_var1 and then the text to the console. 
This is for multithreading bad because you could get fragmented output if the cout is called by multiple threads which could be the case in callbacks.